code/modules/power/floodlight.dm

```diff
@@ -79,8 +79,13 @@
 			setting_text = "standard lighting"
 		if(4)
 			setting_text = "high power"
+			//MonkeStation Edit Start
+			//Moths are flung at lamps
+			Grab_Moths(get_turf(src))
 	if(user)
 		to_chat(user, "You set [src] to [setting_text].")
+		user.changeNext_move(CLICK_CD_MELEE)
+			//MonkeStation Edit End
```

Comment:
I'd rather have the comments ONLY encase what you changed, like this:
```dm
    Grab_moths()//monkestation change
if(-snip-)
    -code-
    user.changeNext_move //Monkestation edit
```
This is just so we KNOW what we changed and what we didn't

Also I hate the fact changeNext_move is inconsistent with how it separates words

Reply:
I can change the first, but not much I can do about ancient inconsistent proc names!